pyquil/paulis.py

```diff
@@ -170,6 +172,24 @@ def __radd__(self, term):
         assert isinstance(term, Number)
         return self + term
 
+    def __sub__(self, term):
+        """Subtracts a PauliTerm from this one.
+
+        :param PauliTerm term: A PauliTerm object
+        :returns: A PauliSum object representing the sum of this PauliTerm and term
+        :rtype: PauliSum
+        """
+        return self + -1. * term
```

Comment:
since this method defines subtraction I need to define it in terms of addition that is already defined